src/NGen.cpp

```diff
@@ -517,7 +518,8 @@ int main(int argc, char *argv[]) {
           // only advance if you would not pass the master next time and the previous layer next time
           if ( layer_next_time <= next_time && layer_next_time <=  prev_layer_time)
           {
-            layer->update_models();
+            if(count%100==0) std::cout<<"Updating layer: "<<layer->get_name()<<"\n";
```

Comment:
Did you intend to keep this line in? No issues with it, just want to make sure.

Reply:
I think we should leave it for now to help indicate what is happening through time as the simulation domain complexity expands.  We may want to configure the interval a bit more flexibly in the future, or even wrap this up in one of the quite build options.